src/AssemblySharedInfoGenerator/AssemblySharedInfo.tt

```diff
@@ -68,7 +68,7 @@ using System.Runtime.InteropServices;
 [assembly: AssemblyFileVersion("<#= this.MajorVersion #>.<#= this.MinorVersion #>.<#= this.BuildNumber #>.<#= this.RevisionNumber #>")]
 <#+
 int MajorVersion = 2;
-int MinorVersion = 1;
+int MinorVersion = 2;
 int BuildNumber = 0;
 int RevisionNumber = ((int)(DateTime.UtcNow - new DateTime(2017,1,1)).TotalDays)*10+((int)DateTime.UtcNow.Hour)/3;
```

Comment:
Since you are updating minor version, can you align this with Dynamo Side: https://github.com/DynamoDS/Dynamo/blob/master/src/AssemblySharedInfoGenerator/AssemblySharedInfo.tt#L75

Reply:
OK, I will align it soon.

Reply:
Thank you!